common/reviews/api/webpack-plugin-utilities.api.md

```diff
@@ -0,0 +1,16 @@
+## API Report File for "@rushstack/webpack-plugin-utilities"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type * as Webpack from 'webpack';
+import type * as Webpack5 from '@rushstack/heft-webpack5-plugin/node_modules/webpack';
```

Comment:
This is a fragile way of referencing this. It'd be cleaner to use package aliases.